remix/app/routes/auth.google.callback.tsx

```diff
@@ -0,0 +1,11 @@
+import type { LoaderArgs } from "@remix-run/node";
+import { authenticator } from "~/config/auth.server";
+
+import { SocialsProvider } from "remix-auth-socials";
```

Comment:
The ordering of imporst should be (third-party first and application code next):

```suggestion
import type { LoaderArgs } from "@remix-run/node";
import { SocialsProvider } from "remix-auth-socials";

import { authenticator } from "~/config/auth.server";
```